tests/smoke-maven.yaml

```
@@ -198,7 +198,7 @@ output:
                   operation: update
                   support_file: false
                   type: file
-            pr-title: Bump spring-boot-starter-parent from 1.5.9.RELEASE to 2.7.2
+            pr-title: Bump org.springframework.boot:spring-boot-starter-parent from 1.5.9.RELEASE to 2.7.2
             pr-body: |
                 Bumps [spring-boot-starter-parent](https://github.com/spring-projects/spring-boot) from 1.5.9.RELEASE to 2.7.2.
```

I think it needs to be changed here too?

---

Ping @yeikel :)

---

I am not sure I understand what you mean @deivid-rodriguez because I made that change with this PR

Can you add a suggestion? 

It is honestly hard to test these

---

Sorry @yeikel. I hear you and I agree, I hope to merge this repo in to core at some point and make working with this easier.

So, what I meant is that the name should also change in the line above?

```suggestion
                Bumps [org.springframework.boot:spring-boot-starter-parent](https://github.com/spring-projects/spring-boot) from 1.5.9.RELEASE to 2.7.2.
```